SwiftWisdom/Core/Foundation/Defaults.swift

```diff
@@ -41,3 +41,14 @@ extension EnumSettingsKeyAccessible {
         return rawValue
     }
 }
+
+extension UserDefaults {
+
+    /// Removes all stored objects from UserDefaults.
+    func ip_clear() {
```

Comment:
Would `removeAllObjects` be clearer, since it's a wrapper around `removeObject` (and also *not* a wrapper around `removePersistentDomain`, `removeVolatileDomain`, `removeSuite`, etc?